frontend/src/components/project/ProjectPageRoot.js

```diff
@@ -355,26 +357,13 @@ export default function ProjectPageRoot({
     toggleShowFollowers();
   });
 
-  const [projectLinkShared, setProjectLinkShared] = React.useState(false);
-  const createShareRecord = (sharedVia) => {
-    if (sharedVia === 8 && projectLinkShared) return; //only create a share-record for the link once per session
-    apiRequest({
-      method: "post",
-      url: "/api/projects/" + project.url_slug + "/set_shared_project/",
-      payload: { shared_via: sharedVia },
-      token: token,
-      locale: locale,
-    })
-      .then(() => {
-        if (sharedVia === 8) {
-          setProjectLinkShared(true);
-        }
-      })
-      .catch(function (error) {
-        console.log(error);
-        if (error && error.reponse) console.log(error.response);
-      });
-  };
+  const apiEndpointShareButton = "/api/projects/" + project.url_slug + "/set_shared_project/";
+  const projectAdminName = project?.creator.name ? project?.creator.name : projectAdmin.name;
+  const projectLinkPath = "/" + locale + "/projects/" + project.url_slug;
```

Comment:
Here I would use the `getLocalePrefix(locale)` function. Otherwise the link will also say `/en` which works but is redundant